src/test/objectstore/test_bluefs.cc

```diff
@@ -135,6 +135,58 @@ TEST(BlueFS, small_appends) {
   rm_temp_bdev(fn);
 }
 
+TEST(BlueFS, very_large_write) {
```

Comment:
This looks fine as a workaround to test the patch quickly.
But IMO there is a couple of general issues with this approach:
1) BlueFS involvement which makes large write occurrence implicit/unclear.
2) This doesn't cover other possible BlockDevice implementations: NVMEDevice/PMEMDevice and non-libaio mode. It's still unclear to me if the latter supports large writes.
I think we should consider making a standalone UT for BlockDevice implementations. May be later though...